src/index.html

```diff
@@ -17,11 +17,389 @@
+
+        <div class="card__rate">
+          <div class="card__rate-stars card__rate-stars--4">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+          </div>
+          <div class="card__rate-text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          <span class="card__button-text">Buy</span>
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="mac photo"
+          class="card__photo"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="card__rate-stars card__rate-stars--4">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+          </div>
+          <div class="card__rate-text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
+        >
+          <span class="card__button-text">Buy</span>
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="mac photo"
+          class="card__photo"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rate">
+          <div class="card__rate-stars card__rate-stars--4">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+          </div>
+          <div class="card__rate-text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="hover"
```

Comment:
The 'data-qa' attribute value is inconsistent with other similar elements. Ensure consistency in naming conventions for QA attributes.